README.md

```diff
@@ -99,3 +127,9 @@ console.log(i)
 
 -- YOUR ANSWER HERE --
 
+```js
+0
+1
+2
+2
+```
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.